protocols/grpc/hyperstart.proto

```diff
@@ -110,20 +113,46 @@ message CreateSandboxRequest {
 message DestroySandboxRequest {
 }
 
+enum IpFamily {
+	v4 = 0;
+	v6 = 1;
+}
+
 message IpAddress {
-	string address = 1;
-	string mask = 2;
+	IpFamily family = 1;
+	string address = 2;
+	string mask = 3;
+}
+
+message Interface {
+	string device = 1;
+	string name = 2;
+	repeated IpAddress ipAddresses = 3;
+	uint64 mtu = 4;
+	string hwAddr = 5;
+}
+
+enum UpdateType {
+	AddIP = 0;
+	DelIP = 1;
+	SetName = 2;
+	SetMtu = 3;
+}
+
+message UpdateInterface {
+	UpdateType type = 1;
+	Interface interface = 2;
+}
+
+message UpdateInterfacesRequest {
+	repeated UpdateInterface updates = 1;
 }
 
-message UpdateInterfaceRequest {
-	uint64 type	=1;
-	string device = 2;
-	string newName = 3;
-	repeated IpAddress ipAddresses = 4;
-	uint64 mtu = 5;
+message InterfacesRequest {
+	repeated Interface interfaces = 1;
```

Comment:
I think it’s not good to handle a bunch of interfaces or routes in single request, it will makes things complicated when part of this request failed, because we need to decides if we should roll back the successful operations or continue with next one. Neither option is good enough

Reply:
@WeiZhang555 Makes sense to me. I'll update the PR.